main.go

```diff
@@ -95,31 +90,31 @@ func main() {
 	}
 
 	if conf.generateGoModels {
-		if err := generateGoModels(conf.modelsPath, conf.goPackagePath, swaggerSpec); err != nil {
+		if err := generateGoModels(*conf.goPackageName, conf.goAbsolutePackagePath, swaggerSpec); err != nil {
```

Comment:
These changes were part of getting the name of the package that the model is being generated for accessible when we generate the code.  Necessary for placing the go.mod file and for later versioning of the models module.